source/simulator/assemblers/stokes.cc

```diff
@@ -207,11 +210,15 @@ namespace aspect
               data.local_rhs(i) += (density * gravity * scratch.phi_u[i])
                                    * JxW;
 
-              if (force != NULL)
+              if (force != NULL && this->get_parameters().enable_additional_stokes_rhs)
                 data.local_rhs(i) += (force->rhs_u[q] * scratch.phi_u[i]
                                       + pressure_scaling * force->rhs_p[q] * scratch.phi_p[i])
                                      * JxW;
 
+              if (elastic_outputs != NULL )
```

Comment:
This implementation means that in the assembly, if the material model fills the elastic outputs, they will be assembled (no matter what the `enable_elasticity` parameter is set to). I just want to make sure that this is really what we want (as for the force terms, they are only assembled if this option is selected in the input file, see above). Shouldn't this rather be 
`if (elastic_outputs != NULL && this->get_parameters().enable_elasticity)`

Reply:
At the moment, the model will just crash if you set `enable_elasticity = false`, and the error message says that you have to switch it on if you want to use the elasticity material model. So even if people don't read the documentation, they will notice. 
This would only be to make sure if people add other material models with elasticity that the elastic terms are only added if `enable_elasticity = true`. 

And I think we agreed that if you want to use the material model without elasticity, you just have to set the elastic moduli to a large value. So no need to do any big changes, just this one line. I think we want to merge this as soon as possible.

Reply:
That's right, there is already an Assert for `Enable elasticity`. Probably a sign that is indeed to time to merge as soon as possible :) I've added a list of future features in a new github issue and will address the remaining points in a little bit.